molo/core/utils.py

```diff
@@ -52,3 +56,21 @@ def generate_slug(text, tail_number=0):
     else:
         # No collisions
         return slug
+
+
+def replace_media_file(file):
+    '''Replace the Current Media Folder.'''
+    media_parent_directory = os.path.dirname(settings.MEDIA_ROOT)
+    zip_file_reference = os.path.join(media_parent_directory, 'new_media.zip')
+
+    if os.path.isdir(settings.MEDIA_ROOT):
+        shutil.rmtree(settings.MEDIA_ROOT)
```

Comment:
This is what I'm concerned about.. did you consider using [copytree](https://docs.python.org/2/distutils/apiref.html#distutils.dir_util.copy_tree)